bin-src/init.ts

```diff
@@ -0,0 +1,144 @@
+import boxen from "boxen";
+import { execaCommand } from 'execa';
+import { findUp } from 'find-up';
+import { writeFile } from 'jsonfile';
+import meow from 'meow';
+import prompts from 'prompts';
+import type { PackageJson } from 'read-pkg-up'
+import readPkgUp from 'read-pkg-up';
+import { getCliCommand, parseNi } from '@antfu/ni';
+
+import noPackageJson from '../node-src/ui/messages/errors/noPackageJson';
+
+export const TestFramework = {
+    STORYBOOK: 'storybook',
+    PLAYWRIGHT: 'playwright',
+    CYPRESS: 'cypress'
+};
+type TestFrameworkType = typeof TestFramework[keyof typeof TestFramework];
+
+export const getPackageManagerInstallCommand = async (args: string[]) => {
+    return await getCliCommand(parseNi, args, { programmatic: true });
+};
+
+const getSBConfigFilePath = async () => {
+    // Walks up directory tree to find nearest Storybook config file.
+    return await findUp(['.storybook/main.ts', '.storybook/main.js', '.storybook/main.tsx', '.storybook/main.jsx', '.storybook/main.mjs', '.storybook/main.cjs'])
+}
+
+export const addChromaticScriptToPackageJson = async ({ packageJson, packagePath }) => {
+    try {
+        const json = {
+            ...packageJson,
+            scripts: {
+                ...packageJson?.scripts,
+                chromatic: `npx chromatic`
+            }
+        }
+        await writeFile(packagePath, json, { spaces: 2 });
+    } catch (e) {
+        console.warn(e)
+    }
+}
+
+export const createChromaticConfigFile = async ({configFile, buildScriptName = null}) => {
+    await writeFile(configFile, {
+        autoAcceptChanges: "main",
+        ...(buildScriptName && {
+            buildScriptName
+        }),
+        onlyChanged: true,
+        skip: "dependabot/**"
+    });
+}
+
+export const installArchiveDependencies = async (packageJson: PackageJson, testFramework: TestFrameworkType) => {
+    let installArgs = ['-D', 'chromatic',`chromatic-${testFramework}`, 'storybook@next', '@storybook/addon-essentials@next', '@storybook/server-webpack5@next']
```

Comment:
Right now, this is intentionally installing SB 8.0 alpha. We can update before release to ensure we are on the latest.

Reply:
We need to check if the user already has SB installed and use the same version they have specified in `package.json` if so.

Reply:
@tmeasday We do this step later on lines 57-60.